hack/test-cmd.sh

```diff
@@ -396,22 +396,29 @@ for version in "${kube_api_versions[@]}"; do
   kube::test::get_object_assert services "{{range.items}}{{$id_field}}:{{end}}" 'kubernetes:kubernetes-ro:redis-master:'
   # Command
   kubectl create -f - "${kube_flags[@]}" << __EOF__
+{
+  "kind": "Service",
+  "apiVersion": "v1beta3",
+  "metadata": {
+    "name": "service-${version}-test"
+  },
+  "spec": {
+    "ports": [
       {
-          "kind": "Service",
-          "apiVersion": "v1beta1",
-          "id": "service-${version}-test",
-          "port": 80,
-          "protocol": "TCP"
+        "protocol": "TCP",
+        "port": 80,
+        "targetPort": 0
```

Comment:
I can't tell what this test is supposed to be doing -- it doesn't look like there are pods backing the service, at least at this point. Does targetPort = 80 work? I believe we should be defaulting targetPort to port now.